### PR TITLE
contentHash is not unique

### PR DESCRIPTION
**Pull request by StanislavBreadless**

### CODE REVIEW

File: docs/api/sdk/js/nfts.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ async mintNFT(mintNft: {
 | Name        | Description                                                                                         |
 | ----------- | --------------------------------------------------------------------------------------------------- |
 | recipient   | the recipient address represented as a hex string                                                   |
-| contentHash | the unique identifier of the NFT represented as a 32-byte hex string (e.g. IPFS content identifier) |
+| contentHash | an identifier of the NFT represented as a 32-byte hex string (e.g. IPFS content identifier) |
```
Comment (popzxc) [inline docs/api/sdk/js/nfts.md:44]: Fmt failures are [still not being reported](https://github.com/matter-labs/zksync-docs/pull/136/checks?check_run_id=3309814969#step:3:211).

Please increase the level of importance of the task.

**Review by popzxc — APPROVED** [review]

**Review by astarinmymind — APPROVED** [review]
lgtm

**Comment by StanislavBreadless** [comment]
@popzxc We'll fix the formatting errors together with the CI (there are too many of them)

**Pull request merged**